common/libs/framing/serial_listeners.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __SERIAL_LISTENERS_H
+#define __SERIAL_LISTENERS_H
+
+enum RxError_t {
```

Comment:
Please rename to simply RxError, make this an `enum class`, and remove the RX_ERROR prefix